server/report_builder/README.md

```diff
@@ -110,7 +110,28 @@ The remote-server needs a store id and a data id to print the report.
 For example, to print a report for a stocktake with id "d734fd45-064e-4ddd-9886-ea71a2797640" from store "80004C94067A4CE5A34FC343EB1B4306":
 
 ```bash
-> report_build print --report output.json --config config.yaml --store-id 80004C94067A4CE5A34FC343EB1B4306 --data-id d734fd45-064e-4ddd-9886-ea71a2797640 --output report_pdf_name.pdf
+> report_build print --report generated/output.json --config config.yaml --store-id 80004C94067A4CE5A34FC343EB1B4306 --data-id d734fd45-064e-4ddd-9886-ea71a2797640 --output report_pdf_name.pdf
+```
+
+### Report templates with arguments
+
+Some reports need additional arguments such as a time range.
+Report arguments are passed on as variable to the used graphql query.
+
+There is an example for a report with arguments in the `example_arguments` directory.
+To specify the arguments create a `argument.json` file in the `report_builder` directory with the content:
+
+```json
+{
+  "creationDatetimeBeforeOrEqual": "2023-01-16T01:35:39.770Z"
```

Comment:
would be helpful to know where this prop name comes from. Is it defined in the report? 
what are the possible variable types? what sort of date format is used?

Reply:
I added some more details about it, thanks!